pulp_2_tests/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -388,30 +388,61 @@ def test_all(self):
         .. _Pulp #1455: https://pulp.plan.io/issues/1455
         .. _Pulp Smash #525: https://github.com/PulpQE/pulp-smash/issues/525
         """
+        task = self.run_task(gen_repo(importer_config={'feed': utils.uuid4()}))
+
+        with self.subTest(comment='check task error description'):
+            self.assertNotEqual(
+                'Unsupported scheme: ',
+                task['error']['description']
+            )
+
+    def test_missing_filelists_error_message(self):
+        """Test whether an error report contains sufficient information.
+
+        Do the following:
+
+        1. Create and sync a repository using a missing filelist feed URL.
+        2. Get a reference to the task containing error information.
+        3. Assert that:
+
+           * The error description is sufficiently verbose. See `Pulp #4262`_
+           * The traceback is non-null. See `Pulp #1455`_.
+
+        .. _Pulp #4262: https://pulp.plan.io/issues/4262
+        """
+        cfg = config.get_config()
+        if cfg.pulp_version < Version('2.19'):
+            raise unittest.SkipTest('This test requires Pulp 2.19 or newer.')
+
+        repo_body = gen_repo(
+            importer_config={'feed': RPM_MISSING_FILELISTS_FEED_URL}
+        )
+        task = self.run_task(repo_body)
+
+        with self.subTest(comment='check task error description'):
+            self.assertIn(
+                "Error retrieving metadata: Error 'Not Found' for",
```

Comment:
I believe looping to ensure all keywords are found here instead of an exact string search may be the current idiot for `stderr` checks. This is to ensure they will not incur future maintenance if the string form or `verbage` changes.

Will let @kersommoura post the correct idiom here, as he had a good working solution.

Reply:
@bherrin3 note that this is not `stderr` it is the message returned by the **tasks api**

Reply:
Main idea is just to look for certain key words, and not compare the whole string. Avoiding failures, if this string is later a bit changed.

https://github.com/PulpQE/Pulp-2-Tests/blob/9fce3e71ab5b2300b99296ca39a34ec183743aba/pulp_2_tests/tests/rpm/api_v2/test_iso_sync_publish.py#L202

A Pulp 3 example.

https://github.com/pulp/pulp_rpm/blob/2c119ef251635ba17e19a501e3b0ba9e3193c086/pulp_rpm/tests/functional/api/test_crud_content_unit.py#L163